src/activities/menu/Menu.qml

```diff
@@ -114,6 +119,10 @@ ActivityBase {
             icon: menuActivity.url + "strategy.svg",
             tag: "strategy"
         },
+        {
+            icon: menuActivity.url + "search-icon.png",
```

Comment:
I  found this on clipart https://openclipart.org/detail/181256/search-icons  ,is this good?
and is there a problem with png format